packages/bot-web-ui/src/components/dashboard/tutorial-tab/quick-strategy-content/quick-strategy-guides-details.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import StrategyDescription from 'Components/quick-strategy/form-wrappers/strategy-description';
+
+type TQuickStrategyGuidesDetail = {
+    tutorial_selected_strategy: string;
+};
+
+const QuickStrategyGuidesDetail = ({ tutorial_selected_strategy }: TQuickStrategyGuidesDetail) => {
+    return (
+        <div>
+            <StrategyDescription tutorial_selected_strategy={tutorial_selected_strategy} />
+        </div>
+    );
+};
```

Comment:
I don't see why we need this as a separate component. Please give me an idea. 🤨
The usage is in one place only and looks like this:
`<QuickStrategyGuidesDetail tutorial_selected_strategy={tutorial_selected_strategy} /> `

Reply:
@maryia-matskevich-deriv we created 1 component which is the StrategyDescription to use in both quick strategy modal description and quick strategy guides in tutorial tab. So this QuickStrategyGuidesDetail component was created with the main purpose of rendering the StrategyDescription component for tutorial tab. So to keep it clear and clean we created this separate component that is related to quick strategy guides for tutorial tab.

Reply:
Yes, but if it performs the same function as component `StrategyDescription`, the only differences are the component name and the wrapper in the div. I'm not sure what the point of it is, isn't it redundant?🫤 
We'll leave that remark to your judgement 👌